react-blogs/Floor-3100/src/components/Card.jsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect } from "react";
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime.js";
+await import('dayjs/locale/es-us.js');
+
+dayjs.extend(relativeTime);
+dayjs.locale('es-us');
+
+// Humanizarfechas
+function humanizarFecha(date) {
+    const wrappedDate = dayjs(date);
+    const daysDiff = Math.abs(wrappedDate.diff(Date.now(), "days"));
+    const currentYear = wrappedDate.year() === dayjs().year();
+    if (!currentYear) {
+        return wrappedDate.format("MMM DD, YYYY")
+    } else if (daysDiff <= 30) {
+        return wrappedDate.fromNow()
+    } else {
+        return wrappedDate.format("MMMM DD")
+    }
+}
+
+const Card = ({ title, desc, img, author, dateTime }) => { 
+
+    useEffect(() => {
+        const timeElement = document.getElementById(dateTime);
+        if (timeElement) {
+            timeElement.innerHTML = "Fecha: " + humanizarFecha(dayjs(timeElement.getAttribute("datetime")));
+        }
+    }, [dateTime]);
```

Comment:
```suggestion
    }, []);
```
Si quieres ejecutar este codigo solo la primera vez que se renderiza el componente usa el array vacio, te aseguras que solo se ejecute una vez.
No uses argumentos como el dateTime aqui, solo estados.